core/templates/dev/head/forms/formBuilder.js

```diff
@@ -598,7 +603,20 @@ oppia.factory('rteHelperService', [
       // Returns a DOM node.
       createRteElement: function(componentDefn, customizationArgsDict) {
         var el = $('<img/>');
-        el.attr('src', componentDefn.iconDataUrl);
+        if (componentDefn.previewDataUrl) {
+          el.attr('src', componentDefn.previewDataUrl);
+        } else if (componentDefn.previewUrlToInterpolate) {
+          el.attr('src',
+            $interpolate(
```

Comment:
Thanks! I tried using that at first, but it seems like line 89 of core/domain/utilities/UrlInterpolationService.js prevents values containing '.' characters which is needed for interpolating file paths.